templates/style/base.scss

```diff
@@ -1,6 +1,56 @@
 // FIXME: Use modules
 @import "vars", "utils", "navbar", "themes", "fa";
 
+/* See FiraSans-LICENSE.txt for the Fira Sans license. */
+@font-face {
+    font-family: 'Fira Sans';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Fira Sans'), url("FiraSans-Regular.woff") format('woff');
```

Comment:
I think it's the most stable part of rustdoc. They were here before I came in and they haven't changed since then. So moving them in a "static" location seems safe enough.

Reply:
@GuillaumeGomez I think pietro means that the files won't appear at all unless you run a build first. It would be nice to be able to use Fira Sans without having to first run a rustdoc build.

Reply:
Oh I see! Sending a PR shortly then.